src/Domain/Analyser.php

```diff
@@ -84,7 +84,7 @@ public function analyse(string $dir, array $files): Collector
     private function analyseFile(Collector $collector, string $filename): void
     {
         $buffer = (string) \file_get_contents($filename);
-        $tokens = @\token_get_all($buffer);
```

Comment:
You should not remove this. 
This has been added because we need to silence the errors which users might have in their code. 

You should instead silence the phpstan message in the phpstan config.